src/interface/raftex.thrift

```diff
@@ -51,6 +51,8 @@ struct AskForVoteRequest {
     5: TermID              term;               // Proposed term
     6: LogID               last_log_id;        // The last received log id
     7: TermID              last_log_term;      // The term receiving the last log
+    8: LogID               committed_log_id;   // Last committed Log ID
+    9: bool                is_transfer;        // Trigger by transfer leader or not
```

Comment:
What's the purpose about 'is_transfer'?

Reply:
There is a corner case, if we start election because of transLeader, the target host's committedLogId would be smaller than old leader's, which would cause old leader `reset`. I have not figured out a better way, do you have any suggestion?

Reply:
Em...   Let me think about it.  It is a read good case.